wheel/src/run_program.rs

```diff
@@ -75,7 +94,7 @@ pub fn run_chia_program(
     args: &[u8],
     max_cost: Cost,
     flags: u32,
-) -> PyResult<(Cost, LazyNode)> {
+) -> PyResult<(PyObject, LazyNode)> {
```

Comment:
this seems risky without also changing the name of the function. The return value will be backwards compatible, so if the python code doesn't update, it might still appear to work, as long as no errors happen